lib/Extension/LanguageServerCodeTransform/LspCommand/GenerateMutatorsCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpactor\Extension\LanguageServerCodeTransform\LspCommand;
+
+use Amp\Promise;
+use Amp\Success;
+use Phpactor\CodeTransform\Domain\Exception\TransformException;
+use Phpactor\CodeTransform\Domain\Refactor\GenerateMutator;
+use Phpactor\CodeTransform\Domain\SourceCode;
+use Phpactor\Extension\LanguageServerBridge\Converter\TextEditConverter;
+use Phpactor\LanguageServerProtocol\ApplyWorkspaceEditResponse;
+use Phpactor\LanguageServer\Core\Command\Command;
+use Phpactor\LanguageServerProtocol\WorkspaceEdit;
+use Phpactor\LanguageServer\Core\Server\ClientApi;
+use Phpactor\LanguageServer\Core\Workspace\Workspace;
+
+class GenerateMutatorsCommand implements Command
+{
+    public const NAME  = 'generate_mutators';
+
+    private GenerateMutator $generateMutator;
+
+    private ClientApi $clientApi;
+
+    private Workspace $workspace;
+
+    public function __construct(
+        ClientApi $clientApi,
+        Workspace $workspace,
+        GenerateMutator $generateMutator
+    ) {
+        $this->generateMutator = $generateMutator;
+        $this->clientApi = $clientApi;
+        $this->workspace = $workspace;
+    }
+
+    /**
+     * @param string[] $propertyNames
+     * @return Promise<ApplyWorkspaceEditResponse|null>
+     */
+    public function __invoke(string $uri, int $startOffset, array $propertyNames): Promise
+    {
+        $textDocument = $this->workspace->get($uri);
+
+        try {
+            $textEdits = $this->generateMutator->generate(
```

Comment:
I guess the API is the same for GenerateAccessor and GenerateMutator? could we create an interface `PropertyAccessGenerator` and then we can use the same infrastructure for both cases?

Reply:
Indeed they're the same. Sure, but should we keep the `GenerateMutator` interface and make it extends `PropertyAccessGenerator`, or should it be removed?

Reply:
If they are exactly the same prob fine to rename the existing one

Reply:
It's done :)